File: 100-Days-Series/Day 56/HollowRectangle.js

// arrow function to print hollow rectangle pattern
const getHollowRectangle=(rows,cols)=>{
    for(let i=1;i<=rows;i++){
        line="";
        for(let j=1;j<=cols;j++){
            if(i==1 || i==rows || j==1 || j==cols){
                line+="* ";
            }
            else{
                line+="  ";
            }
        }
        console.log(line);
    }
}
// Let's Start here
let rows=5;
let cols=10;
getHollowRectangle(rows,cols);;

File: 100-Days-Series/Day 56/HollowRectangle.cpp
#include<iostream>
using namespace std;
// function to print hollow rectangle pattern
void getHollowRectangle(int rows,int cols){
    for(int i=1;i<=rows;i++){
        for(int j=1;j<=cols;j++){
            if(i==1 || rows==i || j==1 || cols==j){
                cout<<"* ";
            }else{
                cout<<"  ";
            }
        }
        cout<<endl;
    }
}
// Let's Start Here
int main(){
    int rows,cols;
    cout<<"\nEnter the number of rows : ";
    cin>>rows;
    cout<<"\nEnter the number of columns : ";
    cin>>cols;
    getHollowRectangle(rows,cols);
    return 0;
}